<ui version="4.0" >
 <author>Reinhold Kainhofer &lt;reinhold@kainhofer.com></author>
 <comment>Configuration page for the print journal mode.</comment>
 <class>CalPrintJournalConfig_Base</class>
 <widget class="QWidget" name="CalPrintJournalConfig_Base" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>311</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>6</number>
   </property>
   <property name="leftMargin" >
    <number>9</number>
   </property>
   <property name="topMargin" >
    <number>9</number>
   </property>
   <property name="rightMargin" >
    <number>9</number>
   </property>
   <property name="bottomMargin" >
    <number>9</number>
   </property>
   <item>
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-weight:600;">Print journal options:&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="mDateRangeGroup" >
     <property name="title" >
      <string>Date &amp;&amp; Time Range</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QRadioButton" name="mAllJournals" >
        <property name="text" >
         <string>&amp;All journal entries</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="mRangeJournals" >
        <property name="focusPolicy" >
         <enum>Qt::TabFocus</enum>
        </property>
        <property name="text" >
         <string>Date &amp;range:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>15</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="mFromDateLabel" >
          <property name="text" >
           <string>&amp;Start date:</string>
          </property>
          <property name="buddy" >
           <cstring>mFromDate</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KPIM::KDateEdit" name="mFromDate" >
          <property name="focusPolicy" >
           <enum>Qt::StrongFocus</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="mToDateLabel" >
          <property name="text" >
           <string>&amp;End date:</string>
          </property>
          <property name="buddy" >
           <cstring>mToDate</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KPIM::KDateEdit" name="mToDate" >
          <property name="focusPolicy" >
           <enum>Qt::StrongFocus</enum>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>324</width>
            <height>21</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>301</width>
       <height>211</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KPIM::KDateEdit</class>
   <extends>QComboBox</extends>
   <header>kdateedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mFromDate</tabstop>
  <tabstop>mToDate</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
